src/model.ts

```diff
@@ -132,6 +132,12 @@ export class JobsModel extends VDomModel {
   private _createJobModel: ICreateJobModel;
   private _listJobsModel: IListJobsModel;
   private _jobDetailModel: IJobDetailModel;
+  /**
+   * Callback that gets invoked whenever a model is updated. This should be used
+   * to call `ReactWidget.renderDOM()` to synchronously update the VDOM rather
+   * than triggering an async VDOM update via Lumino. See #34.
```

Comment:
```suggestion
   * than triggering an async VDOM update via Lumino.
```

Not necessary to refer to issues or PRs in code comments

Reply:
Why not? Issue IDs are immutable and can add a lot of valuable context that can't fit inside of a code comment.

Reply:
I left a comment on that issue that describes in detail why this issue occurs and why this workaround is necessary. That level of detail is valuable for future devs.

Reply:
The problem is that "See #34" outside the context of a GitHub issue or PR just means "see number thirty-four." Especially as code gets copied and reused, this will mean less over time.

You could provide a more obvious cue with a URL, although I've never found it popular to include URLs in source code.

Reply:
I think this comment is sufficiently obvious to developers. Using my discretion here and merging.